docs/user_manual/type_data.md

```diff
@@ -0,0 +1,514 @@
+---
+layout: page
+title: "5. Type Data"
+section: 5
+---
+
+
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Reference Genomes" %}
+
+Reference genome refers to a file that sequenced data will be aligned to. This is set at the project level, and can be
+changed on both the Create Project and Edit Project page.
+
+Reference genomes must be added to the MISO database directly. You should talk to a MISO administrator if you require a
+new one.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Sample Types" %}
+
+Sample type is defined by [NCBI](https://www.ncbi.nlm.nih.gov/) and is required in order to submit samples to the SRA
+(see the [SRA section](sra.html)). The values available in MISO should be limited to the default options, which are the
+same as defined in the SRA schema.
+
+It is possible to archive sample types if they are not used within your organization. An archived sample type will not
+show up in the sample type options when creating or editing samples, unless the samples being edited have already used
+that sample type. Sample types can always be unarchived if they are needed in the future. Archiving sample types can
+only be done in the MISO database directly, so you should talk to a MISO administrator about archiving any unused
+sample types. Unarchiving sample types must similarly be done via direct database access.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Sample Classes and Categories" %}
+
+Note: These items only apply if your site uses [detailed sample mode](site_configuration.html#detailed_sample_mode).
+
+Detailed sample mode introduces a hierarchy of samples. Sample classes identify where a sample fits into the hierarchy.
+Sample categories are used to group sample classes into levels. Sample categories may not be modified. See
+[Detailed Sample Mode](site_configuration.html#detailed_sample_mode) for more information about the sample categories
+and ghost samples.
+
+{% assign figure = 1 %}
+Sample classes, and the relationships between them, can be customized to suit your requirements. Figure {{figure}}
+shows the sample hierarchy used at OICR.
+
+{% include userman-figure.md num=figure cap="OICR sample hierarchy" img="type-data-sample-hierarchy.png" %}
+
+Sample classes and valid relationships can only be modified via direct access to the MISO database, and so must be done
```

Comment:
Can you explain "valid relationships" a bit further here? I'd particularly want to see a bit about how they're used to define a custom hierarchy, and a definition of "parenting" as it's used a lot in the bullet points below.

Reply:
I think it's also worth explaining class vs category in detail here.

Reply:
A previous paragraph contains "Sample classes identify where a sample fits into the hierarchy. Sample categories are used to group sample classes into levels," and the diagram helps to illustrate this. I added more description of sample classes and relationships. Are there any other details regarding categories that would be useful?

Reply:
Could you explain what something being  "parented to a class" means?

Reply:
Do you mean beyond what I added already?

> Sample relationships have a parent and child class. This means that if you have a sample of the parent class, you can propagate a sample of the child class. In figure {{figure}}, the relationships are illustrated using arrows that point from the child class to the parent class.

Reply:
Ugh sorry, I hadn't seen that. That looks good!